frontends/common/constantFolding.cpp

```diff
@@ -519,7 +519,7 @@ const IR::Node* DoConstantFolding::postorder(IR::Slice* e) {
     const IR::Expression* msb = getConstant(e->e1);
     const IR::Expression* lsb = getConstant(e->e2);
     if (msb == nullptr || lsb == nullptr) {
-        ::error(ErrorType::ERR_EXPECTED, "%1%: bit indexes must be compile-time constants", e);
+        ::error(ErrorType::ERR_EXPECTED, "%1%: bit indices must be compile-time constants", e);
```

Comment:
@mbudiu-vmw I have checked over the rest of this PR quite carefully, and the rest of it only adds a test program with STF file that looks good to me.  I wanted to call your attention to this change in case you had any comment for/against it.  I believe Hemant considers this an English spelling fix, but if 'indexes' appears in a bunch of other places already, it might as well be a 'standard alternate spelling' as far as p4c is concerned.